src/group-constructors.js

```diff
@@ -19,7 +19,50 @@
   }
 
   function constructor(children, timingInput) {
+    if (this._parent === undefined)
+      this._parent = null;
     this.children = children || [];
+    for (c in this.children) {
+      this.children[c]._parent = this;
+    }
+    this._isAncestor = function(effect) {
+      var a = this._parent;
```

Comment:
do you also want to check whether `effect == this`? It looks like I could add an `Animation` to be a child of itself.